src/Symfony/Component/Console/Command/Command.php

```diff
@@ -677,6 +678,8 @@ public function getUsages(): array
     /**
      * Gets a helper instance by name.
      *
+     * @return HelperInterface
```

Comment:
could be real?

Reply:
No, that's a BC break. This will trigger a deprecation notices for child classes and then we can apply this in the next major version.

Reply:
not in a minor version